cluster-up/cluster/kind/podpreset.sh

```diff
@@ -0,0 +1,48 @@
+ #!/usr/bin/env bash
+
+set -xe
+
+source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
+
+function pod_preset::enable_admission_plugin() {
+    local -r cluster_name=$1
+
+    docker exec "$cluster_name-control-plane" bash -c 'sed -i \
+    -e "s/NodeRestriction/NodeRestriction,PodPreset/" \
+    -e "/NodeRestriction,PodPreset/ a\    - --runtime-config=settings.k8s.io/v1alpha1=true" \
+    /etc/kubernetes/manifests/kube-apiserver.yaml'
+}
+
+function pod_preset::validate_admission_plugin_is_enabled() {
+    local -r wait_time=$1
+    local -r admission_plugin=$2
+
+    timeout "${wait_time}s" bash <<EOT
+function is_admission_plugin_enabled() {
+    grep -Po "kube-apiserver.*--enable-admission-plugins=.*\K${admission_plugin}" <<< "$(ps -ef)"
+}
+until is_admission_plugin_enabled; do
+    sleep 1
+done
+EOT
+    [ "$?" -ne 0 ] && echo "FATAL: failed to enable admission plugin $admission_plugin" && return 1
+}
+
+function pod_preset::create_virt_launcher_fake_product_uuid_podpreset() {
+    local -r namespace=$1
+
+    if ! _kubectl get ns "$namespace" &>2; then
+        _kubectl create ns "$namespace"
+    fi
+
+    _kubectl apply -f "$KIND_MANIFESTS_DIR/product-uuid-podpreset.yaml" -n "$namespace"
+}
+
+function pod_preset::expose_unique_product_uuid_per_node() {
+    local -r cluster_name=$1
+    local -r namespace=$2
+
+    pod_preset::enable_admission_plugin "$cluster_name"
+    pod_preset::validate_admission_plugins_is_enabled "10" "PodPreset"
```

Comment:
It fails because the `s` of plugins

Reply:
Good catch, I changed it to consider the right container according to cluster name